pages/program-details/[pid].tsx

```diff
@@ -21,33 +26,34 @@ export const ProgramDetails: React.FC = () => {
         ["/api/program/" + pid, pid],
         fetchWithId,
     );
-    if (classListError || programInfoError) {
-        return (
-            <Text>
-                An error has occurred.{" "}
-                {classListError
-                    ? classListError.toString()
-                    : programInfoError.toString()}
-            </Text>
-        );
+
+    if (isLoading || loading) {
```

Comment:
If you go with above then
```suggestion
    if (isMeLoading || isSessionLoading) {
```